src/Symbolics.jl

```diff
@@ -15,6 +15,8 @@ using Reexport
 
 using DomainSets
 
+using Setfield
```

Comment:
This needs to be added to the Project.toml

Reply:
Actually wait, it's already in there, which is why tests pass. Is it imported somewhere? Or was that some old deps mistake that's an accidental fix now?

Reply:
maybe a reexport thing? not sure